src/Component/CheckOut.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+
+class CheckOut extends React.Component{
+    constructor(){
+        super()
+        this.state = {
+            firstName: '',
+            lastName: '',
+            email: '',
+            creditCard: '',
+            zipCode: ''
+        }
+    }
+    allFieldsValid = () => {
+        const { firstName, lastName, email, creditCard, zipCode } = this.state;
+        return firstName && lastName && email && creditCard && zipCode;
+      }
+    handleSubmit = (e) =>{
+        e.preventDefault()
+        if (this.allFieldsValid()) {
+            alert(`Yey, Purchase was complete! You will be charged $`);
+          } else {
+            alert(`Item is not valid`)
+          }
```

Comment:
Add logic in this function to check that the credit card number and zip code are the right length.